conda-lock.yml

```diff
@@ -5955,31 +6030,21 @@ package:
     sha256: a4166e3d8ff4e35932510aaff7aa90772f84b4d07e9f6f83c614cba7ceefe0eb
   category: main
   optional: false
-- name: nvidia-nvcomp-cu12
-  version: 4.2.0.14
-  manager: pip
-  platform: linux-64
-  dependencies: {}
-  url: https://files.pythonhosted.org/packages/e5/58/371b6df7e9a86921324f1cc780a86c3e47ecbe52f4cc1148efceb4f102cf/nvidia_nvcomp_cu12-4.2.0.14-py3-none-manylinux_2_28_x86_64.whl
-  hash:
-    sha256: 0d9bc07bf63aeae2e9877d34c8aab6781cf26efa8d2fec05af8b0ec58ca1fd41
-  category: main
-  optional: false
 - name: zarr
-  version: 3.0.4.dev4+ga8c0db34
+  version: 3.0.5.dev161+gf89b2324
   manager: pip
   platform: linux-64
   dependencies:
     donfig: '>=0.8'
     numcodecs: '>=0.14'
-    numpy: '>=1.25'
+    numpy: '>=1.26'
     packaging: '>=22.0'
     typing-extensions: '>=4.9'
-  url: git+https://github.com/akshaysubr/zarr-python.git@a8c0db34ef488d654bac92ee5805bbffd2d7d8dc
+  url: git+https://github.com/akshaysubr/zarr-python.git@f89b232486d040047364acfbb6d3825b8b6a2a6b
```

Comment:
https://github.com/zarr-developers/zarr-python/pull/2863 should be close to merging, so can also wait a bit to remove this last pip-installed package from the environment.yml.